crates/tbdex/src/messages/quote.rs

```diff
@@ -94,7 +94,7 @@ pub struct QuoteDetails {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub fee: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub payment_instructions: Option<PaymentInstructions>,
```

Comment:
PaymentInstructions eventually this will be called `PaymentInstruction`?

Reply:
oh nice, good catch yeah updating that now...